test_main.py

```diff
@@ -5,22 +5,49 @@
 Description: This program will display a member's bank account summary including balances and subshares
 """
 
-from main import Account
+from main import Account, Checking, Credit
 
 
 def test_account_number():
     """
     Test that the account number is correct
     """
-    test_member = Account("246812", "Test McTest")
+    test_member = Account("1111111", "Test McTest", "$1.00")
     summary = test_member.account_summary()
-    assert "246812" in summary
+    assert "1111111" in summary
 
 
 def test_name():
     """
     Test that the account number is correct
     """
-    test_member2 = Account("3691113", "Testianna Testerson")
+    test_member2 = Account("2222222", "Testianna Testerson", "$2.00")
     summary2 = test_member2.account_summary()
     assert "Testianna Testerson" in summary2
+
+
+def test_balance():
+    """
+    Test that the account blance is correct
+    """
+    test_member3 = Account("3333333", "Testian Testoppolus", "$3.00")
+    summary3 = test_member3.account_summary()
+    assert "$3.00" in summary3
+
+
+def test_checks():
+    """
+    Test that the checking account has checks
+    """
+    test_member4 = Checking("4444444", "Test Test", "$4.00", 8)
+    summary4 = test_member4.account_summary()
+    assert "8" in summary4
+
+
+def test_credit_card():
+    """
+    Test that the account has a credit card
+    """
+    test_member5 = Credit("5555555", "Testian Testoppolus", "$5.00", 5)
+    summary5 = test_member5.account_summary()
+    assert "5" in summary5
```

Comment:
## Week 9 Grading

| Points | Location        | Criteria                                |
| ------ | --------------- | --------------------------------------- |
| 5      | main.py         | 2 Classes extend the parent class       |
| 10     | main.py         | 2 Functions on the extended classes     |
| 5      | main.py         | Functions set a class attribute         |
| 10     | pytest coverage | Tests cover all aspects of main.py      |
| 10     | pylint          | Pylint score >= 8.0.                    |
|        | instructor      | Code style, efficiency, and readability |
| 40/40 |